src/components/FilmTile.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+export class FilmTile extends React.PureComponent {
+    constructor(props) {
+        super (props);
```

Comment:
If  you use props in your component, you should describe all of them: specify types and default values using "propTypes" and "defaultProps" properties.
https://codeburst.io/why-you-should-use-proptypes-in-react-native-e6f5ef78e7dd